sound/soc/intel/boards/sof_sdw_rt_sdca_jack_common.c

```diff
@@ -152,6 +152,9 @@ int sof_sdw_rt_sdca_jack_exit(struct snd_soc_card *card, struct snd_soc_dai_link
 	if (!ctx->headset_codec_dev)
 		return 0;
 
+	if (!SOF_JACK_JDSRC(sof_sdw_quirk))
+		return 0;
```

Comment:
oh, isn't that a bug you want to correct before moving the code to a different file? That would make backports easier if indeed this is a problem

Reply:
> oh, isn't that a bug you want to correct before moving the code to a different file? That would make backports easier if indeed this is a problem

ack